integrations/completions/ort-completion.fish

```diff
@@ -25,7 +25,7 @@ complete -c ort -n "__fish_seen_subcommand_from advise" -l output-dir -s o -r -F
 complete -c ort -n "__fish_seen_subcommand_from advise" -l output-formats -s f -r -fa "JSON YAML" -d 'The list of output formats to be used for the ORT result file(s).'
 complete -c ort -n "__fish_seen_subcommand_from advise" -l label -s l -r -d 'Set a label in the ORT result, overwriting any existing label of the same name. Can be used multiple times. For example: --label distribution=external'
 complete -c ort -n "__fish_seen_subcommand_from advise" -l resolutions-file -r -F -d 'A file containing issue and rule violation resolutions.'
-complete -c ort -n "__fish_seen_subcommand_from advise" -l advisors -s a -r -d 'The comma-separated advisors to use, any of [NexusIq, OssIndex, Osv, VulnerableCode].'
+complete -c ort -n "__fish_seen_subcommand_from advise" -l advisors -s a -r -d 'The comma-separated advisors to use, any of [NexusIQ, OSSIndex, OSV, VulnerableCode].'
```

Comment:
I wonder whether it's really worth to allow having custom plugin IDs. They are IDs after all, so to some extend an implementation detail, and in cases there the ID needs to be specified by the user, like a CLI option, the comparison is case-insensitive anyway.

Reply:
> I wonder whether it's really worth to allow having custom plugin IDs.

I was thinking of the `AudiOpenSourceDiagnosticsReporter` class which uses `AOSD` as `type`/`id`. I like that the class has a readable name but for a plugin id that's too long.

> They are IDs after all, so to some extend an implementation detail, and in cases there the ID needs to be specified by the user, like a CLI option, the comparison is case-insensitive anyway.

Using `OSV` instead of `Osv` is just cosmetic, but why not if the effort is minimal? We could also go the opposite direction and make them all lowercase.

Reply:
> Using `OSV` instead of `Osv` is just cosmetic, but why not if the effort is minimal?

I'm thinking about maintenance effort / future technical debt here. But the change indeed is so simple that this should not be an issue.

Reply:
> I was thinking of the `AudiOpenSourceDiagnosticsReporter` class which uses `AOSD` as `type`/`id`.

BTW, speaking of AOSD, just as a heads-up, I'm planning to address https://github.com/oss-review-toolkit/ort/issues/9063 soon.